packages/cspell-lib/src/lib/textValidation/lineValidatorFactory.ts

```diff
@@ -314,6 +315,11 @@ export function lineValidatorFactory(sDict: SpellingDictionary, options: Validat
         }
 
         function _checkPossibleWords(possibleWord: TextOffsetRO): ValidationIssue[] {
+            // If unknown words should be completely ignored, return empty array
+            if (unknownWords === 'ignore-all') {
```

Comment:
Flagged words need to be reported, so returning here effectively turns off the spell checker.